Main page changes

asu21244 opened the pull request:
- Added a search bar in the landing page
- Added a whats sets us apart
- made the src folder more organized
- Made the logo appear

Zineb2a reviewed (changes requested):
I've had a look at the recent changes in the pull request, and here are a few observations and suggestions:

1. Login Button : I noticed the login button isn't clickable anymore.
2. Navigation Bar : The navbar seems to be fixed now. Was this an intentional change?, Besides that, well done with the logo
3. CTA Placement : The CTA seems to have shifted. Do you think we can adjust its positioning for consistency?
4. Search Bar Padding : Perhaps adding some padding to the left and right of the search bar might enhance its look. 
5. Responsiveness:
   - The "What Sets Us Apart" section seems to face some responsiveness issues. 
   - The search Bar responsiveness is good. GG

asu21244 commented:
- personally on my side when I first looked at the landing page before adding my changement, the log in button was never clickable, only the sign up button was. The only thing I changed about the log in button was the background color cause when the nav bar would go all the way down and the page was white, the log in button wasnt visible.
- For the “what sets us apart”, I’ll check why it’s not responsive.
- For the CTA, the second I added the search bar it automatically shifted which means there’s something wrong in the CTA code.

asu21244 commented:
- the search bar is now to low
- CTA still needs to be fixed
- logo is way better
- navbar doesn't move anymore which is good & the log in button is clickable
